OutputNumbers.java

```diff
@@ -0,0 +1,11 @@
+class OutputNumbers {
+  public static void main(String[] args) {
+    for (int i = 0; i <= 100; i++){
+      if(i % 3 == 0) {
+        System.out.print("3*" + i/3 + "\n");
```

Comment:
use println method instead of ugly "\n" - it can cause crossplatform issues due to the different line-ending conventions. println will care